alerts/cluster-network-operator/OVNKubernetesNorthdInactive

```diff
@@ -0,0 +1,56 @@
+# OvnKubernetes Northd Inactive Alert
+
+## Meaning
+
+The alert `OVNKubernetesNorthdInactive` fires when there are no active
+instances of Northd within the high availabilty set.
+
+## Impact
+
+ovn-northd is a centralized daemon that translates the logical network flows from
+the OVN Northbound Database into the physical datapath flows in the
+OVN Southbound database. If there are no active instances of ovn-northd, then
+this action will not occur, which will cause a degraded network.
+Existing workloads should remain stable, but any attempt at altering the
+network control plane will fail.
+
+### Fix alerts before continuing
```

Comment:
since you started this we add this line to each alert that has dependent alerts:

## Fix alerts before continuing
Resolve any alerts that may cause this alert to fire: [Alert hierarchy](https://github.com/openshift/runbooks/blob/master/alerts/cluster-network-operator/hierarchy/alerts-hierarchy.svg)